samples/LoggingTracer/LoggingTracer.AspNetCore/LoggingTracer.AspNetCore.csproj

```diff
@@ -0,0 +1,21 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
```

Comment:
perhaps the name of the project should be `LoggingTracer.Demo.AspNetCore`? I was confused by this name initially

Reply:
ok, done